profiles/common/modules/custom/multisite_drupal_toolbox/multisite_drupal_toolbox.module

```diff
@@ -1657,3 +1656,54 @@ function multisite_drupal_toolbox_features_post_restore($op, $items) {
     }
   }
 }
+
+/**
+ * Remove PHP Warning logs from HTML5 during a loadHTML.
+ *
+ * Extend SimpleXMLElement function.
+ *
+ * @see $dom->loadHTML;
+ */
+function multisite_drupal_toolbox_load_html(&$dom, $html) {
```

Comment:
You shouldn't pass $dom by reference here. An object is always passed by its object identifier and thus the reference is not needed. See http://php.net/manual/en/language.oop5.references.php